dev/com.ibm.ws.rest.handler.config_fat/fat/src/com/ibm/ws/rest/handler/config/fat/FATSuite.java

```diff
@@ -23,11 +23,13 @@
 
 @RunWith(Suite.class)
 @SuiteClasses({
-                ConfigRESTHandlerAppDefinedResourcesTest.class,
-                ConfigRESTHandlerJCATest.class,
-                ConfigRESTHandlerJMSTest.class,
-                ConfigRESTHandlerTest.class,
-                ConfigOpenApiSchemaTest.class
+                // ConfigRESTHandlerAppDefinedResourcesTest.class,
+                // ConfigRESTHandlerJCATest.class,
+                // ConfigRESTHandlerJMSTest.class,
+                ConfigRESTHandlerAuditTest.class,
+                ConfigRestHandlerAuditFeatureTest.class,
+                // ConfigRESTHandlerTest.class,
+                // ConfigOpenApiSchemaTest.class
```

Comment:
Instead of commenting out tests, if you only want to run a specific test, you may want to switch to using `-Dfat.test.class.name=<class name>` as discussed in [the Open Liberty wiki](https://github.com/OpenLiberty/open-liberty/wiki/FAT-Tests#to-run-only-a-single-test-class-in-a-fat-use-the-command)